test/tasks/StateOverrideManager.t.sol

```diff
@@ -210,29 +213,42 @@ contract StateOverrideManagerUnitTest is Test {
         MultisigTask task = createAndRunTask(fileName, SECURITY_COUNCIL_CHILD_MULTISIG);
 
         uint256 expectedNonce = task.nonce();
-        uint256 expectedTotalOverrides = 3; // i.e. (default + 2 user defined)
+        uint256 expectedTotalOverrides = 4; // i.e. (2 default + 2 user defined)
         Simulation.StateOverride[] memory allOverrides =
-            assertDefaultStateOverrides(expectedNonce, expectedTotalOverrides, task);
+            assertDefaultStateOverrides(expectedNonce, expectedTotalOverrides, task, SECURITY_COUNCIL_CHILD_MULTISIG, 0);
         assertEq(
-            allOverrides[1].overrides[0].key, overrideKey, "First address user override key must match expected value"
+            allOverrides[2].overrides[0].key, overrideKey, "First address user override key must match expected value"
         );
         assertEq(
-            allOverrides[1].overrides[0].value, bytes32(uint256(9999)), "First address user override must be applied"
+            allOverrides[2].overrides[0].value, bytes32(uint256(9999)), "First address user override must be applied"
         );
         assertEq(
-            allOverrides[2].overrides.length,
+            allOverrides[3].overrides.length,
             1,
             "Second address is not the parent multisig so it should only have 1 override"
         );
         assertEq(
-            allOverrides[2].overrides[0].key, overrideKey, "Second address user override key must match expected value"
+            allOverrides[3].overrides[0].key, overrideKey, "Second address user override key must match expected value"
         );
         assertEq(
-            allOverrides[2].overrides[0].value, bytes32(uint256(8888)), "Second address user override must be applied"
+            allOverrides[3].overrides[0].value, bytes32(uint256(8888)), "Second address user override must be applied"
         );
         vm.removeFile(fileName);
     }
 
+    function testOnlyParentOverridesAppliedWhenChildMultisigIsNotSet() public {
+        vm.createSelectFork("sepolia");
+        string memory nonNestedSafeToml = "l2chains = [{name = \"Base Sepolia Testnet\", chainId = 84532}]\n" "\n"
+            "templateName = \"DisputeGameUpgradeTemplate\"\n" "\n"
+            "implementations = [{gameType = 0, implementation = \"0x0000000FFfFFfffFffFfFffFFFfffffFffFFffFf\", l2ChainId = 84532}]\n";
+        string memory fileName = createTempTomlFile(nonNestedSafeToml);
+        MockDisputeGameTask dgt = new MockDisputeGameTask();
+        dgt.simulateRun(fileName);
+        uint256 expectedNonce = dgt.nonce();
+        assertDefaultStateOverrides(expectedNonce, 1, dgt, address(0), 0);
+        vm.removeFile(fileName);
+    }
```

Comment:
I don't really understand how this test is testing that `OnlyParentOverridesAppliedWhenChildMultisigIsNotSet`, can you add comments explaining what we are doing here?

Reply:
Implemented here: https://github.com/ethereum-optimism/superchain-ops/pull/771/commits/2979b3c74d89bba4ca8fb0c6774bb122400ecad3